torchvision/utils.py

```diff
@@ -42,14 +42,19 @@ def make_grid(tensor, nrow=8, padding=2):
     return grid
 
 
-def save_image(tensor, filename, nrow=8, padding=2):
+def save_image(tensor, filename, nrow=8, padding=2, range=None):
     """
     Saves a given Tensor into an image file.
     If given a mini-batch tensor, will save the tensor as a grid of images.
     """
     from PIL import Image
     tensor = tensor.cpu()
     grid = make_grid(tensor, nrow=nrow, padding=padding)
-    ndarr = grid.mul(255).byte().transpose(0,2).transpose(0,1).numpy()
+    if range is not None:
+        assert isinstance(tuple, range), "range has to be a tuple (min, max) if specified"
```

Comment:
I think the arguments in here are inverted, and should be instead `isinstance(range, tuple)`